src/linux_c.zig

```diff
@@ -418,7 +418,7 @@ pub extern fn sysinfo(__info: [*c]struct_sysinfo) c_int;
 
 pub fn get_free_memory() u64 {
     var info: struct_sysinfo = undefined;
-    if (sysinfo(&info) == @as(c_int, 0)) return @as(u64, @bitCast(info.freeram)) *% @as(c_ulong, @bitCast(@as(c_ulong, info.mem_unit)));
+    if (sysinfo(&info) == @as(c_int, 0)) return @as(u64, @bitCast(info.freeram +% info.bufferram)) *% @as(c_ulong, @bitCast(@as(c_ulong, info.mem_unit)));
```

Comment:
Looks like what libuv does here is first attempt to read `/proc/meminfo` and if that fails, it fallback to the previous codepath. It does not use `buferram`. We probably need to do what libuv does here to be consistent with Node

https://github.com/nodejs/node/blob/main/deps/uv/src/unix/linux.c#L1828-L1842

Reply:
What's the best way to test this code path on OSX? Would I just spin up a VM with its own dev env and then build it there